mixer/docker/Dockerfile.mixer

```diff
@@ -1,16 +1,23 @@
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=default
 
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
 # The following section is used as base image if BASE_DISTRIBUTION=default
-# No tag available https://hub.docker.com/_/scratch?tab=description
-# hadolint ignore=DL3006
-FROM scratch as default
+FROM docker.io/istio/base:${BASE_VERSION} as default
+
 # obtained from debian ca-certs deb using fetch_cacerts.sh
 ADD ca-certificates.tgz /
 
+# Sudoers used to allow debug utilities.
+RUN useradd -m --uid 1337 istio-proxy && \
+    echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers
+USER istio-proxy
+
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
 # hadolint ignore=DL3007
-FROM gcr.io/distroless/static:latest as distroless
+FROM gcr.io/distroless/static:nonroot as distroless
```

Comment:
I'm curious, how does distroless handles the user (1337) ? If it works without adduser - why do we add user in the other distros ? Just for sudo ?

Reply:
the user (1337) won't be created when using distros image.